src/Choice Tile/index.js

```diff
@@ -0,0 +1,62 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { SIZES } from '../constants';
+import { wcBool } from '../utils';
+
+const ChoiceTile = ({
+  checked,
+  children,
+  className,
+  disabled,
+  icon,
+  id,
+  invalid,
+  onChange,
+  size,
+  subdued,
+  title,
+  ...rest
+}) => {
+  return (
+    <label className={classNames({ hxChoice: true, className })}>
```

Comment:
non blocking: a potential shorter way to say the same thing:
```suggestion
    <label className={classNames( 'hxChoice', className)}>
```

Reply:
I think this alternative would add a class name of undefined when there is no `className`, right?

Reply:
only if it was written like this: `classNames={`hxChoice  ${className}`}` but `className()` does not include undefined values. 

from the docs: https://github.com/JedWatson/classnames#readme
```
// other falsy values are just ignored
classNames(null, false, 'bar', undefined, 0, 1, { baz: null }, ''); // => 'bar 1'
```